slim/slim/CVE-2015-2171.yaml

```diff
@@ -0,0 +1,8 @@
+title:     PHP object injection attack vulnerability in Slim.
+link:      https://github.com/slimphp/Slim/issues/1034
+cve:       CVE-2015-2171
+branches:
+    2.x:
+        time:     2015-03-01,
```

Comment:
comma should be removed

Reply:
And you should add the exact date (with time in UTC) for when the fix was merged